src/llama_stack/core/datatypes.py

```diff
@@ -376,6 +377,22 @@ class VectorStoresConfig(BaseModel):
         default=None,
         description="Default embedding model configuration for vector stores.",
     )
+    default_query_expansion_model: QualifiedModel | None = Field(
```

Comment:
maybe just stick to one of expansion or rewrite across the board? its a little confusing if we have expansion here but rewrite elsewhere